src/store/MarkerStore.js

```diff
@@ -13,7 +13,7 @@ export default class MarkerStore {
     children;
     x;
     y;
-    @observable display;
+    @observable display = 'none';
```

Comment:
MarkerStore is deprecated, we will remove it in next release.
Deriv.app has already migrated to the new FastMarker.